.ci/toolchain-vs2019-amd64-facebook.yml

```diff
@@ -0,0 +1,79 @@
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - .ci/azure-pipelines-aws-agent.yml
+      - .ci/templates/toolchain.yml
+trigger:
```

Comment:
Do we want to do a `schedules` trigger with `*/90 * * * *` to trigger this every 90 minutes?